event_track_assistant/models/event.py

```diff
@@ -278,14 +278,21 @@ class EventTrackPresence(models.Model):
 
     @api.depends('session', 'session.allowed_partner_ids')
     def _compute_allowed_partner_ids(self):
-        for presence in self:
-            presence.allowed_partner_ids = (
-                [(6, 0, presence.session.allowed_partner_ids.ids)])
-            if self.env['marketing.config.settings']._get_parameter(
-               'show.all.customers.in.presences'):
-                customers = self.env['res.partner'].search([])
-                for customer in customers:
-                    presence.allowed_partner_ids = [(4, customer.id)]
+        try:
+            users = self.env.ref('event.group_event_manager').users
+        except Exception:
+            pass
+        show_all = self.env['marketing.config.settings']._get_parameter(
+            'show.all.customers.in.presences')
+        if (show_all and show_all.value == 'True' and
```

Comment:
if (show_all and show_all.value == 'True' and self.env.user.id in users.ids):
    allowed_ids = self.env['res.partner'].search([])
else:
     allowed_ids = self.filtered(lambda x: x.state == 'pending').mapped('session.allowed_partner_ids.ids'):
presence.allowed_partner_ids = ([(6, 0, allowed_ids)])